content/docs/self-hosted-runners.md

```diff
@@ -202,22 +202,22 @@ env:
 
 ## Using on-premise machines as self-hosted runners
 
-You can also use `cml-runner` to set up a local self-hosted runner on your local
+You can also use `cml-runner` to set up a self-hosted runner on your local
 machine or on-premise GPU cluster.
 
 ```dvc
 cml-runner \
   --repo="$repository_url" \
   --token="$personal_access_token" \
-  --labels="custom,runner" \
-  --idle-timeout=180
+  --labels="local,runner" \
+  --idle-timeout=0
```

Comment:
seems odd to have `0`. Maybe leave out?

Reply:
I remember bringing this up before. Really seems like a slightly broken CLI API - should maybe fix in https://github.com/iterative/cml?

Reply:
https://github.com/iterative/cml/pull/700

Reply:
ok but still

```suggestion
```

the idea is that the default is computed and probably sensible (before and after https://github.com/iterative/cml/pull/700)

Reply:
continuing in #93

Reply:
The default value is intended for cloud runners; please refer to the commit message body on 2cbaafaa2a57b055faa1fb274000fc6d49f793e2

Reply:
When running this locally, users (read: myself at least) would probably want the runner to keep listening forever and not to die after 300 seconds of inactivity.

Reply:
> The default value is intended for cloud runners; 

Who says 🙂 ? My own gpus stays until there is not more jobs in the pipeline. Then I shut down the machine via script. Its very energy unfriendly.

Reply:
> _Who says 🙂 ? My own gpus stays until there is not more jobs in the pipeline. Then I shut down the machine via script. Its very energy unfriendly._

Alright! 😄  To my mind, it makes more sense for cloud runners than for on-premises machines, but your approach seems excellent both for the planet and for the budget.

What if we recommend no timeput by default and add a callout bellow?

> 🌳  Note: you can set `--idle-timeout=<seconds>` so `cml runner` exits after `<seconds>` of inactivity, and, perhaps, configure your computer to shutdown automatically afterwards

Reply:
Yeah suggesting 0 sounds good to me